presto-orc/src/main/java/io/prestosql/orc/reader/TimestampColumnReader.java

```diff
@@ -55,6 +59,7 @@
     private final OrcColumn column;
 
     private long baseTimestampInSeconds;
+    private Optional<LongUnaryOperator> timestampConversion;
```

Comment:
We generally don't use functional style programming in these readers because it hurts performance.  I'm not sure it will matter here because the timestamp conversions are computationally intensive, but I would have written this with two loops for each usage instead.